playbook.yaml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Configure App
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: TRELLO_API_KEY
+      prompt: Trello API Key?
+
+    - name: TRELLO_API_TOKEN
+      prompt: Trello API Token?
+
+    - name: TRELLO_BOARD_ID
+      prompt: Trello Board ID?
+      default: 65e4d3b571aacb487f9dc1bf
+
+    - name: TRELLO_TO_DO_LIST_ID
+      prompt: Trello "To-Do" List ID?
+      default: 65e4d3b571aacb487f9dc1c6
```

Comment:
Nice, I'm a fan of extracting these for future flexibility, whilst also putting in defaults for the non-sensitive ones so that we don't have to type them every time!